platform.md

```diff
@@ -400,6 +402,7 @@ Usage:
 | `<app>`          | `CNB_APP_DIR`          | `/workspace`                                           | Path to application directory                                                                                                                                |
 | `<build-config>` | `CNB_BUILD_CONFIG_DIR` | `/cnb/build-config`                                    | Path to build config directory                                                                                                                               |
 | `<buildpacks>`   | `CNB_BUILDPACKS_DIR`   | `/cnb/buildpacks`                                      | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout))                                                               |
+| `<exec-env>`     | `CNB_EXEC_ENV`         | `production`                                           | Target execution environment. Standard values: "production", "test", "development".                                                                           |
```

Comment:
This all seems fine to me, but I'm looking at this [comment](https://github.com/buildpacks/rfcs/pull/274/files#r1165993054) from the RFC and I don't see that specified anywhere. Are we expecting the lifecycle to skip over buildpacks or groups that don't match the current environment?

Reply:
yes. the question is if we want to skip an entire group if one of the buildpack doesn't match the env. We need @hone to chime in there